formula-android/src/main/java/com/instacart/formula/fragment/FormulaFragment.kt

```diff
@@ -28,28 +28,30 @@ class FormulaFragment : Fragment(), BaseFormulaFragment<Any> {
         arguments!!.getParcelable<FragmentKey>(ARG_CONTRACT)!!
     }
 
-    private lateinit var fragmentEnvironment: FragmentEnvironment
-    internal var viewFactory: ViewFactory<Any>? = null
+    internal lateinit var fragmentEnvironment: FragmentEnvironment
+    internal lateinit var viewFactory: ViewFactory<Any>
```

Comment:
Since these are internal properties, I'm just going to set them directly in `FragmentFlowRenderView`. Updated `fragmentEnvironment` to follow the same pattern.